src/components/scenes/FioAddressDetailsScene.js

```diff
@@ -7,7 +7,7 @@ import IonIcon from 'react-native-vector-icons/Ionicons'
 
 import s from '../../locales/strings.js'
 import { ConnectWalletsConnector as ConnectWallets } from '../../modules/FioAddress/components/ConnectWallets'
-import { bundlesIsAlmostSpent, findWalletByFioAddress, noBundles } from '../../modules/FioAddress/util'
+import { BUNDLES_AMOUNT_ALERT, findWalletByFioAddress } from '../../modules/FioAddress/util'
```

Comment:
Please change all variables that have `bundles` to `bundledTxs` or `bundled_txs` bundles is not a proper name